MonoGame.Framework/Random.cs

```diff
@@ -0,0 +1,386 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace Microsoft.Xna.Framework
+{
+    public static partial class MathHelper
+    {
+        /// <summary>
+        /// Class for generating random numbers consistently when using Mono or Microsoft compilers.
+        /// </summary>
+        public class Random
```

Comment:
The source file should have the standard MonoGame header block.  See https://github.com/mono/MonoGame/blob/develop/MonoGame.Framework/Utilities/Hash.cs#L1 for an example.